recipes/geoalchemy2/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "GeoAlchemy2" %}
```

Comment:
Names for both conda and PyPI are case insensitive. We prefer the lower case name to avoid issues with the feedstock. The directory must be identical to the package name for the feedstock creation.